tools/macos/tweak-dockermachine.sh

```diff
@@ -26,12 +33,17 @@ sudo route -n -q delete 172.17.0.0/16
 sudo route -n -q add 172.17.0.0/16 $MACHINE_VM_IP
 netstat -nr |grep 172\.17
```

Comment:
move this block to the end and add an `echo Adding route forwarding on your host machine, enter sudo password if/when prompted`.